packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-modify-instance-fleet-by-name.ts

```diff
@@ -24,7 +24,7 @@ export interface EmrModifyInstanceFleetByNameProps {
    *
    * @see https://docs.aws.amazon.com/emr/latest/APIReference/API_InstanceFleetModifyConfig.html
    *
-   * @default None
+   * @default - None
    */
   readonly targetOnDemandCapacity: number;
```

Comment:
Same here about the default. What does this actually mean? None is very confusing.

Same comment for other number fields throughout

Reply:
fair point, all I was *trying* to do was introduce the hyphen in the PR, but i'll just re-hash them all. 
essentially, the general callout is that default docs are inadequate/confusing. 

I'll also add it to the tracking issue since this doesn't seem to be something we're consistently raising the bar on across modules and PRs